src/Microsoft.AspNet.Razor/Generator/LiteralAttributeCodeGenerator.cs

```diff
@@ -62,9 +63,9 @@ public override bool Equals(object obj)
         {
             var other = obj as LiteralAttributeCodeGenerator;
             return other != null &&
```

Comment:
Did we not want to ensure symmetric `Equals`?In order to do that we should be calling down into base.equals